fishtest/tests/test_rundb.py

```diff
@@ -9,6 +11,11 @@ class CreateRunDBTest(unittest.TestCase):
 
   def setUp(self):
     self.rundb = util.get_rundb()
+    self.rundb.runs.ensure_index(
```

Comment:
Just a question: why defined both here and in `create_indices.py`?

Reply:
this index is on the test db `fishtest_test` which `create_indexes.py` doesn't yet touch (maybe it should). using a separate db is to prevent unexpected behavior for happening in case someone runs the test script from a dev/prod environment.

this `ensure_index` call is needed in this test because we're explicitly using this `finished_ltc_runs` index in `get_finished_runs()` and without ensuring the index here that call would fail

Reply:
Ah, I missed that it was in the testing code. Sorry for the noise.